testing/jest/button.tests.tsx

```diff
@@ -414,4 +440,43 @@ describe('Button', () => {
 
         expect(tree.is('.dx-button')).toBe(true);
     });
+
+    describe('DefaultOptionRules', () => {
+        function getDefaultProps() {
+            defaultOptions({
+                device: () => false,
+                options: {},
+            });
+            return Button.defaultProps;
```

Comment:
It's very bed using global functions and global variables. Can we rewrite it to pure function?

Reply:
No, We can't